src/StackAdmin/Azs.Fabric.Admin/Tests/src/Common.ps1

```diff
@@ -1,24 +1,27 @@
 ﻿
+$global:SkippedTests = @(
+    'TestInfraRoleInstancePowerOn',
```

Comment:
@deathly809 why are we skipping these tests?

Reply:
I think when we were testing we had a bug on the RP side which would crash the service.